### PR TITLE
feat(AddressComponent): allow modifying a specific address label

### PR DESCRIPTION
**Pull request by th0rOdinson**

### CODE REVIEW

File: apps/root/src/hooks/useLabelHandler.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
Comment (FiboApe) [inline apps/root/src/hooks/useLabelHandler.ts:1]: Love this label handler!

File: apps/root/src/services/labelService.ts [hunk 2]
```diff
@@ -32,19 +32,59 @@ export default class LabelService {
...
+
+  async deleteLabel(labeledAddress: string): Promise<void> {
+    const user = this.accountService.getUser();
+    if (!user) {
+      return;
+    }
+    const currentLabels = this.labels;
+    if (currentLabels.hasOwnProperty(labeledAddress)) {
+      const deletedLabel = this.labels[labeledAddress];
+      delete currentLabels[labeledAddress];
```
Comment (FiboApe) [inline apps/root/src/services/labelService.ts:75]: should this actually be inside the try? since we want to revert all that when it fails right?

**Review by FiboApe — APPROVED** [review]
Great work!

File: apps/root/src/services/labelService.ts (hunk 2 above)
Comment (th0rOdinson) [reply apps/root/src/services/labelService.ts:75]: You're right, it makes the code clearer to move those lines into the `try` block. Even though they're unlikely to fail due to the `hasOwnProperty` check, I think it's a good practice. I'll move them. Thanks for the feedback!

File: apps/root/src/hooks/useLabelHandler.ts (hunk 1 above)
Comment (th0rOdinson) [reply apps/root/src/hooks/useLabelHandler.ts:1]: Thank you sir

**Pull request merged**